CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ Upgrade to React v15.2.1 if you are affected. (#4729)
 - [React] Fix more invalid props warning (#4667, #4675, #4685, #4725)
 - [Snackbar] Change the action's PropType to node (#4716)
 - [TextField] False should be a valid value (#4728)
+- [BottomNavigation] Initial implementation (#4846)
```

Comment:
It's not longer at the right position. You can remove it or moving it under the `HEAD` section.